### DISCUSSION TITLE
硬體高手必備程式語言，成為科技趨勢「整合者」

### DISCUSSION THREAD

**Discussion by 239216** [1101-GS4719-C/Python-Community-GS4719-C#43] [General]
侯永清特別指出，現在的硬體產業人才，最少需要通曉一種軟體程式語言。科技發展如此快速，10 年之後，「會寫程式」也許就像現在「會上網搜尋」一樣，變成基本能力。最近他才要求部門裡的所有成員，重新複習及翻修一遍程式語言，「程式語言是一種思考方式，不只能訓練思考，也幫助大家增加生產力。」「程式語言的概念很相似，學會一種之後，其他的也能類比貫通，」侯永清表示。
吳田玉表示，半導體以人才為本，目前，台灣半導體的人才總量明顯不足，「尤其在新世代的語言能力、軟體能力、遠端應對能力，如何因應保護主義、平行世界、遠端連結的挑戰，將是新的命題。」
新聞來源:https://technews.tw/2021/02/15/programming-language-becomes-a-technological-trend/

**Comment by jerry850313** [comment]
程式語言不只是軟體語言還有硬體語言，
根據文中公司的分類來看，應該更偏向硬體和韌體。
硬體的寫法跟軟體不太一樣，
硬體更接近電路(邏輯閘)，而軟體比較像是電腦指令的組合。

**Reply by 239216** [reply]
了解! 原來還有這樣的區分
